wsdl/ver10/device/wsdl/devicemgmt.wsdl

```diff
@@ -336,6 +336,11 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 						<xs:documentation>Indicates maximum number of storage configurations supported.</xs:documentation>
 					</xs:annotation>
 				</xs:attribute>
+				<xs:attribute name="StorageConfigurationRenewal" type="xs:boolean">
```

Comment:
Should this be **StorageConfigurationCredentialRenewal** as storage configuration is quite a big entity holding lot of configuration fields?

Reply:
The renewal includes the server uris also, so it felt a bit larger than just credentials. But I'm open to it